pyanaconda/modules/storage/fcoe/fcoe.py

```diff
@@ -60,6 +60,40 @@ def write_configuration(self, sysroot):
         log.debug("Write FCoE configuration to %s.", sysroot)
         fcoe.write(sysroot)
 
+    def get_nics(self):
+        """Get all NICs.
+
+        :return: a list of names of network devices connected to FCoE switches
+        """
+        return [nic for nic, dcb, auto_vlan in fcoe().nics]
+
+    def get_dracut_arguments(self, nic):
+        """Get dracut arguments for the given FCoE device.
+
+        :param nic: a name of the network device
+        :return: a list of dracut arguments
+
+        FIXME: This is just a temporary method taken from blivet.
+        """
+        dcb = True
+
+        for _nic, _dcb, _auto_vlan in fcoe().nics:
+            if nic == _nic:
+                dcb = _dcb
+                break
+        else:
+            return list()
+
+        if dcb:
+            dcb_opt = "dcb"
+        else:
+            dcb_opt = "nodcb"
```

Comment:
This could be rewritten as ``dcb_opt = "dcb" if dcb else "nodcb"``.

Reply:
Fixed.